src/components/Sidebar/index.jsx

```diff
@@ -52,9 +52,18 @@ class Sidebar extends React.Component {
       enableSubscriptionManagementScreen,
       enableSamlConfigurationScreen,
       enableAnalyticsScreen,
+      enableLearnerPortal,
       enableLmsConfigurationsScreen,
     } = this.props;
 
+    // Hide Settings link if there are no visible tabs
+    const shouldShowSettingsLink = (
+      features.SETTINGS_PAGE && (
+        enableLearnerPortal || features.FEATURE_SSO_SETTINGS_TAB
+       || (features.EXTERNAL_LMS_CONFIGURATION && features.SETTINGS_PAGE_LMS_TAB && enableLmsConfigurationsScreen)
+      )
+    );
```

Comment:
gonna leave this one as is since we'd have to pass in all the variables to the util function. We'll eventually be getting rid of this check once lms configuration tabs are ready :).